src/Liip/RMT/VCS/Git.php

```diff
@@ -47,6 +47,10 @@ public function getTags()
 
     public function createTag($tagName)
     {
+        // this requires git and gpg configured
+        if (isset($this->options['sign-tag']) && $this->options['sign-tag']) {
+            return $this->executeGitCommand("tag -s $tagName -m $tagName");
+        }
```

Comment:
doesn't really matter - however this slightly changes previous behaviour (as this will now always add $tagName as a comment to the Tag (signed tags require that for some reason), which previously was not the case. (It shouldn't break anything though)